src/k8s/pkg/utils/cidr.go

```diff
@@ -19,3 +20,20 @@ func GetFirstIP(subnet string) (net.IP, error) {
 	r = append(make([]byte, 16), r...)
 	return net.IP(r[len(r)-16:]), nil
 }
+
+// GetKubernetesServiceIPsFromServiceCIDRs returns a list of the first IP addrs from a given service cidr string.
+func GetKubernetesServiceIPsFromServiceCIDRs(serviceCIDR string) ([]net.IP, error) {
+	var firstIPs []net.IP
+	cidrs := strings.Split(serviceCIDR, ",")
+	if v := len(cidrs); v != 1 && v != 2 {
+		return nil, fmt.Errorf("invalid ServiceCIDR value: %v", cidrs)
+	}
+	for _, cidr := range cidrs {
+		ip, err := GetFirstIP(cidr)
+		if err != nil {
+			return nil, fmt.Errorf("could not get IP from CIDR: %v", cidr)
```

Comment:
let's wrap the error that explains what happened
```suggestion
			return nil, fmt.Errorf("could not get IP from CIDR %q: %w", cidr, err)
```